web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -390,6 +390,82 @@ $.extend(fixmystreet.set_up, {
     });
   },
 
+  manage_duplicates: function() {
+      // Deal with changes to report state by inspector/other staff, specifically
+      // displaying nearby reports if it's changed to 'duplicate'.
+      function refresh_duplicate_list() {
```

Comment:
These functions will be being created on each call to `manage_duplicates`, but this can probably wait for a tidy after the current batch are merged.